book/plugins/android/single-bundle.md

```diff
@@ -0,0 +1,35 @@
+## Single Bundle React Naitve
+
+![single-bundle.png](./single-bundle.png)
+
+### OVERVIEW
+Single bundle is a plugin that makes multiple react native components implementation stable, increase it's performance and it is more efficient and easy to handle in a native app. This plugin gives ability to change version of react native and react. It also aggregate all react native plugins into one single bundle and upload it to S3. Plugins becomes modules under the names specified as plugin identifiers from plugin configuration.
```

Comment:
I this we should remove the `it's`I would write something like: `Single bundle is a plugin that makes multiple react native components implementation stable, increase performance and allows and easy and efficient way to handle these components within a native app.